new-pipeline/style/dashboards.css

```diff
@@ -128,7 +128,7 @@
   justify-content: center;
   align-items: center;
   text-align: center;
-  margin-bottom: 3em;
+  margin-bottom: 1em;
```

Comment:
How do things look with `margin-bottom: 0;` ? That'll keep the plot at the same scroll index as it used to be.

Reply:
I actually can't check this out... The graphs don't load in my browser but they do in my phone. I wasn't able to judge by it...